src/common/function/src/scalars/geo/measure.rs

```diff
@@ -46,10 +46,7 @@ impl Function for STDistance {
 
     fn signature(&self) -> Signature {
         Signature::new(
-            TypeSignature::Exact(vec![
-                ConcreteDataType::string_datatype(),
-                ConcreteDataType::string_datatype(),
-            ]),
+            TypeSignature::Exact(vec![DataType::Utf8, DataType::Utf8]),
```

Comment:
it would be better and more generic to map string data type to {`utf8`|`large utf8`|`utf8 view`}, as well as bytes type

Reply:
Will datafusion do auto coerce for utf8/utf8view? I can call a udf defined in utf8 with a string literal

Reply:
maybe no as I know, they will be boiled down to different concrete arrow array types, a nightmare to udf

Reply:
there is something like string array accessor, but it can't handle everything